### DISCUSSION TITLE
react example not working with version 0.9.0

### DISCUSSION THREAD

**Discussion by krzysztofpniak** [perspective-dev/perspective#1468] [Q&A]
### Steps to Reproduce:
download tag v0.9.0
run following:
`cd examples/react` 
`yarn && yarn start`

### Expected Result:
application will launch

### Actual Result:
$ webpack serve --open
[webpack-cli] Failed to load '/Users/krzysztofpniak/js/perspective/examples/react/webpack.config.js'
[webpack-cli] Error: Cannot find module '@finos/perspective/dist/esm/config'

### Environment:
macos big sur, nodejs 14

### Additional Context:
version v0.8.0 works

**Comment by texodus** [comment]
Please review the [Perspective developer's guide](https://perspective.finos.org/docs/md/development.html#building), and perhaps also [Yarn's Getting Started](https://yarnpkg.com/getting-started) guide, as you'll need to install, configure and build the project as well.

> yarn start react

**Reply by krzysztofpniak** [reply]
Have you checked  that it works in isolated environment? Anyways, why I need to build the project to just use library from npm?

**Reply by texodus** [reply]
I don't know what an "isolated" environment is, but the examples only run in the `yarn workspace` they are embedded in, as intended.  This is explained clearly in the [Yarn Workspaces Documentation](https://yarnpkg.com/features/workspaces).

It will probably install outside of the workspace as well, but it sounds like you've already `yarn`'d the project root which symlinks these to `packages/*`, which you have not built.

**Reply by krzysztofpniak** [reply]
I have created an "isolated" environment:
https://codesandbox.io/s/small-framework-0e9c7?file=/src/App.js

please guide me what should I do to get it working.

**Comment by kryaksy** [comment]
I've got similar problem here in my project. I've tried to check it in an "isolated" environment, like @krzysztofpniak, and still get errors. Here:

https://stackblitz.com/edit/perspective-poc-2lsdng?file=src%2FApp.jsx

Run `yarn serve` after loaded.

**Reply by krzysztofpniak** [reply]
@texodus are there any chances for a fix?

**Reply by zemeolotu** [reply]
@krzysztofpniak @kryaksy got the react example working in a stand alone [repo](https://github.com/zemeolotu/perspective-react-typescript-example) if you want to check it out.

@texodus Whats do you think about having a separate repos for examples? It's not always obvious what's wrong when you try to build the examples outside the perspective repo. I think thats mostly because of the transient dependencies and configs from the other packages. My main concern with this  would be the examples getting stale. What do you think?

**Reply by kryaksy** [reply]
> @krzysztofpniak @kryaksy got the react example working in a stand alone [repo](https://github.com/zemeolotu/perspective-react-typescript-example) if you want to check it out.
> 
> @texodus Whats do you think about having a separate repos for examples? It's not always obvious what's wrong when you try to build the examples outside the perspective repo. I think thats mostly because of the transient dependencies and configs from the other packages. My main concern with this would be the examples getting stale. What do you think?

I cloned your project. It works. Thank you! 
I think I understood the problem. I don't know why should I, but I didn't exclude monaco editor in this webpack rule:
` {
        test: /\.css$/,
        exclude: [/node_modules\/monaco-editor.*/],
        use: ["style-loader", "css-loader"],
    }`

I will try it in poc and the project.
